main.scss

```diff
@@ -19,6 +19,10 @@
 	.o-teaser--small {
 		@include oTeaserSmall;
 
+		&.o-teaser--has-image {
```

Comment:
As mentioned in the other comments, can these styles be added to teasers with the video styles